paper-radio-group.html

```diff
@@ -26,94 +30,135 @@
 
 @group Paper Elements
 @element paper-radio-group
-@extends core-selector
-@homepage github.io
 -->
 
-<link rel="import" href="../core-a11y-keys/core-a11y-keys.html">
-<link rel="import" href="../core-selector/core-selector.html">
-<link rel="import" href="../paper-radio-button/paper-radio-button.html">
+<dom-module name="paper-radio-group">
+  <style>
+    :host {
+      display: inline-block;
+    }
 
-<polymer-element name="paper-radio-group" extends="core-selector" role="radiogroup">
+    iron-selector ::content > * {
+      padding: 12px;
+    }
+  </style>
 
   <template>
+    <iron-selector selected="{{selected}}" attr-for-selected="name"
+        selectable="paper-radio-button">
+      <content id="items" select="*"></content>
+    </iron-selector>
+  </template>
+
+</dom-module>
+
+<script>
+  Polymer({
+    is: 'paper-radio-group',
 
-    <core-a11y-keys target="{{}}" keys="up left" on-keys-pressed="{{selectPrevious}}"></core-a11y-keys>
-    <core-a11y-keys target="{{}}" keys="down right" on-keys-pressed="{{selectNext}}"></core-a11y-keys>
-  
-    <style>
-    
-      :host {
-        display: inline-block;
+    hostAttributes: {
+      role: 'radiogroup'
```

Comment:
Should I do that for `paper-radio-button` and `paper-checkbox` too? I've left those with the default role already.